### DISCUSSION TITLE
either i'm blind or the instructions to download were not clear

### DISCUSSION THREAD

**Discussion by Vikingrad** [BepInEx/BepInEx#942] [General]
i have no idea how to install this I can't even find anything online plz help

**Comment by Lishenntser** [comment]
You just put it in the game then, run it, and It will create folders where you can add plugins lol

**Reply by Randomcooldud** [reply]
he means he cant find the file download
im also having the same problem

**Reply by Lishenntser** [reply]
Front page
Releases
Download the one for your user case
